### ISSUE TITLE
The dropdown arrow in the theme option is not aligned correctly

### ISSUE THREAD

**Issue by mukul314** [Clueless-Community/seamless-ui#2193]
## Current behaviour
When we change the theme from light to dark, the dropdown arrow is not aligned correctly and it looks messy.

## Expected behaviour
The arrow should be visible clearly

## Screenshot 

<img width="258" alt="2023-04-30 15_49_43-Breadcrumb Component – SeamLess UI - Brave" src="https://user-images.githubusercontent.com/76463001/235348049-62add1eb-9273-4804-818b-2573ae9b2183.png">

**Comment by simplysabir** [comment]
Hey, Can i work on this issue?
Here's what i've found. There's issue with Padding and Positioning and I Have Found the Solution.
I believe I have a solution to this issue that I'd like to work on. Would it be possible to assign the issue to me so that I can start working on it?
Can you please assign it to me?

Here's what i've come up with
![image](https://user-images.githubusercontent.com/76485932/235350473-8c1709e7-ecb5-491a-8dbc-fecb16f5bc52.png)

**Comment by Rajdip019** [comment]
Sure work on it, If possible give some more padding on x-axis. @simplysabir

**Comment by simplysabir** [comment]
Sure, On it.

**Comment by simplysabir** [comment]
@Rajdip019 can you pls guide me to find the code. i did the changes using developer mode.

now i'm not able to find the code at this repo :
https://github.com/Clueless-Community/seamless-ui

**Comment by simplysabir** [comment]
Hey @Rajdip019 , I'm not able to find the dropdown component/part in the codebase. I'm Using `website-dev` branch.
could you pls help me out?

**Comment by Lakhaninawaz** [comment]
Can i work it on this issue

**Comment by simplysabir** [comment]
Hey Bro, I have the solution for this. You can try on other issues as there are more available.

**Comment by simplysabir** [comment]
Yes, I'm on it. But I'm not able to find the dropdown component/part in the codebase. I'm Using `website-dev` branch.
could you pls help me out?

**Comment by Rajdip019** [comment]
@RavindraP04 can you help him with this?

**Comment by RavindraP04** [comment]
@simplysabir This dropdown is by default available by Nextra themes, Checkout this https://nextra.site/docs/docs-theme/theme-configuration#dark-mode-and-themes

**Comment by simplysabir** [comment]
Okay lemme see

**Comment by simplysabir** [comment]
Hey @RavindraP04 , Went through this and codebase again thoroghly. still haven't found it. were you able to?

**Comment by RavindraP04** [comment]
@simplysabir I am also trying to figure out how to solve it, 
found this website which is also made using Nextra https://swr.vercel.app/docs/getting-started
In this website, there is custom dropdown for languages and beside that there is option for theme, I am looking in their codebase to know how they have done it, you can also check it.

**Comment by simplysabir** [comment]
@RavindraP04 any updates? i wasn't able to find that

**Comment by RavindraP04** [comment]
> @RavindraP04 any updates? i wasn't able to find that

I stopped looking for it due to exams, but will let you know if I find anything

**Comment by simplysabir** [comment]
Hey, I was still looking for it but couldn't figured it out. secondly, I have used nextra several times for documentation but not a single time this type of problem has occur. so I'm also wondering why did this occur?
lemme know if you've found anything.
